scaleapi/__init__.py

```diff
@@ -781,3 +782,46 @@ def import_file(self, file_url: str, **kwargs) -> File:
         payload = dict(file_url=file_url, **kwargs)
         filedata = self.api.post_request(endpoint, body=payload)
         return File(filedata, self)
+
+    def create_benchmark(
+        self,
+        task_type: TaskType,
+        project: str,
+        expected_response: Dict,
+        initial_response: Optional[Dict] = None,
+        **kwargs,
+    ) -> Benchmark:
+        """This method can only be used for Self-Serve projects.
+        Supported Task Type: [
+            ImageAnnotation,
+            Categorization,
+            TextCollection,
+            NamedEntityRecognition
+        ]
+        Parameters may differ based on the given task_type.
+
+        Args:
+            task_type (TaskType):
+                Task type to be created
+                i.e. `TaskType.ImageAnnotation`
+            **kwargs:
+                Passing in the applicable values into thefunction
+                definition. The applicable fields and further
+                information for each task type can be found in
+                Scale's API documentation.
+                https://docs.scale.com/reference
```

Comment:
Should we just change this since this isn't actually true right now? Or leave it in for now because it will be true later on?